<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Первый список</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Второй список</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConvert_to_binary_1"/>
    <addaction name="actionRead_from_binary_1"/>
    <addaction name="separator"/>
    <addaction name="actionConvert_to_binary_2"/>
    <addaction name="actionRead_from_binary_2"/>
    <addaction name="separator"/>
    <addaction name="actionSave_current_operation_result_to_Result_txt"/>
    <addaction name="actionClose_the_project"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Operations with structures</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUnion"/>
    <addaction name="actionIntrersection"/>
    <addaction name="actionDifference"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Sort</string>
    </property>
    <addaction name="actionSort_by_numbers"/>
    <addaction name="actionSort_by_groupes_and_names"/>
    <addaction name="actionSort_up_by_grade"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_text_file">
   <property name="text">
    <string>Open text file</string>
   </property>
  </action>
  <action name="actionopen_text_file">
   <property name="text">
    <string>считать входной текстовый файл со структурами Stud1</string>
   </property>
  </action>
  <action name="actionopen_binary_file">
   <property name="text">
    <string>открыть бинарный файл для записи структур типа Stud1</string>
   </property>
  </action>
  <action name="actionconvert_data_from_to_text_to_binary">
   <property name="text">
    <string>конвертировать данные из текстового формата в бинарный</string>
   </property>
  </action>
  <action name="actionread_binary_file">
   <property name="text">
    <string>считать бинарный файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Объединение</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Пересечение</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Разность</string>
   </property>
  </action>
  <action name="actionConvert_to_binary_1">
   <property name="text">
    <string>Convert to binary 1</string>
   </property>
  </action>
  <action name="actionRead_from_binary_1">
   <property name="text">
    <string>Read from binary 1</string>
   </property>
  </action>
  <action name="actionConvert_to_binary_2">
   <property name="text">
    <string>Convert to binary 2</string>
   </property>
  </action>
  <action name="actionRead_from_binary_2">
   <property name="text">
    <string>Read from binary 2</string>
   </property>
  </action>
  <action name="actionUnion">
   <property name="text">
    <string>Union</string>
   </property>
  </action>
  <action name="actionIntrersection">
   <property name="text">
    <string>Intrersection</string>
   </property>
  </action>
  <action name="actionDifference">
   <property name="text">
    <string>Difference</string>
   </property>
  </action>
  <action name="actionSort_by_numbers">
   <property name="text">
    <string>Sort by book numbers</string>
   </property>
  </action>
  <action name="actionSort_by_groupes_and_names">
   <property name="text">
    <string>Sort by groupes (in groupes by names)</string>
   </property>
  </action>
  <action name="actionSave_current_operation_result_to_Result_txt">
   <property name="text">
    <string>Save current results to Result.txt</string>
   </property>
  </action>
  <action name="actionClose_the_project">
   <property name="text">
    <string>Close the project</string>
   </property>
  </action>
  <action name="actionSort_up_by_grade">
   <property name="text">
    <string>Sort down by grade (with the same grade by names)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
